.devcontainer/Dockerfile

```
@@ -0,0 +1,96 @@
+FROM ubuntu:24.04
+
+USER root
+
+RUN apt-get update --quiet && \
+    export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install --quiet --yes \
+        apt-transport-https \
+        apt-utils \
+        sudo \
+        git \
+        less \
+        wget \
+        curl \
+        tree \
+        graphviz \
+        software-properties-common && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Apptainer (Singularity)
+RUN add-apt-repository -y ppa:apptainer/ppa && \
+    apt-get update --quiet && \
+    apt install -y apptainer && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set up directories
+RUN mkdir -p /workspaces/.nextflow
+
+# Install Seqera Platform "Tower Agent"
+RUN curl -fSL https://github.com/seqeralabs/tower-agent/releases/latest/download/tw-agent-linux-x86_64 > tw-agent && \
+    chmod +x tw-agent && \
+    mv tw-agent /usr/local/bin/tw-agent
+
+# Uncomment if we need to pin the Nextflow version
+ENV NXF_EDGE=0
+ENV NXF_VER=24.10.4
+ENV NXF_HOME=/workspaces/.nextflow
+
+ENV NFNEURO_RAW_REPOSITORY="https://raw.githubusercontent.com/scilus/nf-neuro/main"
+ENV NFCORE_MODULES_GIT_REMOTE=https://github.com/scilus/nf-neuro.git
+ENV NFCORE_MODULES_BRANCH=main
+ENV NFCORE_SUBWORKFLOWS_GIT_REMOTE=https://github.com/scilus/nf-neuro.git
+ENV NFCORE_SUBWORKFLOWS_BRANCH=main
+
+# Install conda
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
```

Forgot this one. You want to append your ARG value to PATH right ? This is how it should be written.

```suggestion
ARG INPATH
ENV PATH="${INPATH:-/root/miniconda3/bin}:$PATH"
```